Makefile

```diff
@@ -135,10 +139,122 @@ proto-gen: install-proto-bin install-license-bin
 	@echo Generating protobuf files
 	PACKAGE=$(m3db_package) $(auto_gen) $(proto_output_dir) $(proto_rules_dir)
 
+.PHONY: install-m3x-repo
+install-m3x-repo: install-glide install-generics-bin
```

Comment:
same as above, might be better to use the version of m3x in `m3db/vendor`

Reply:
Same response as first response.